src/journeys_utils.js

```diff
@@ -410,20 +411,23 @@ journeys_utils.finalHookups = function(branchViewData, storage, cta, banner, hid
 	Array.prototype.forEach.call(actionEls, function(el) {
 		el.addEventListener('click', function(e) {
 			cta();
+			journeys_utils.refToBranchObj._publishEvent('didClickJourneyCTA');
 			journeys_utils.animateBannerExit(banner);
 		})
 	})
 	var cancelEls = doc.querySelectorAll('.branch-banner-continue');
 	Array.prototype.forEach.call(cancelEls, function(el) {
 		el.addEventListener('click', function(e) {
 			journeys_utils.animateBannerExit(banner);
+            journeys_utils.refToBranchObj._publishEvent('didClickJourneyContinueCTA');
```

Comment:
I would remove 'CTA' from the end of this so it doesn't get confused with the actual CTA